### ISSUE TITLE
4.0.4-3 Doesn't let you unlock area

### ISSUE THREAD

**Issue by lkgregg** [PokeAlert/PokeAlert#283]
I have "locked" area while I'm in it so it doesn't always center on where I'm actually standing but when I go back to try to turn off lock all it gives me is option to lock at a different radius or area with any accounts that aren't already "locked" on an area.  I had to "delete" my accounts then re-add them to be able to unlock an area.   It use to unlock the area I am currently in but now it doesn't.

**Comment by PokeAlert** [comment]
Tap on the grey circles my friend

**Comment by lkgregg** [comment]
You mean the one right below the lock symbol that starts/stops the automatic scans?  Just started the app this morning, didn't pick a locked area today but the lock symbol is showing for 430 m, I had set it around my house yesterday before I sent this.  The area around my house is gray showing it is locked, I want to go to another area, I click the lock button, which use to unlock the area, now all it does is bring up account selection.  When I hit the continuous scan it does the gray area only, even though my open circle is outside that area, which is what I would expect, I turn off cont scan move my clear circle a block or two away and press lock button, it comes up with area 0/0 group: none, radius: 430m, random number of accounts/select all and nothing in list. Save nothing gray still over house area not where my circle is at now.  If I try to do just single fast scan when I have locked area the list of account to select is blank because they are all on the locked area.  I tried clicking a variety of things before submitting this issue.
